light_curves/code_src/fluxconversions.py

```diff
@@ -1,14 +1,44 @@
+import numpy as np
 from acstools import acszpt
 from astropy.time import Time
 
 
-def convert_wise_flux_to_millijansky(nanomaggy_flux):
+def convert_wise_flux_to_millijansky(flux_nanomaggy, *, band=None):
```

Comment:
I'm cleaning up my github backlog, and wonder why not use astropy and its unit equivalencies instead of coding up the flux--mag conversion here?

Reply:
Perhaps we should review all the functions in fluxconversions.py to see whether we can use Astropy methods instead. Adding this to #177.